prow/flagutil/string_slice.go

```diff
@@ -0,0 +1,50 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package flagutil
+
+import (
+	"strings"
+)
+
+// StringSlice is a generic command-line flag for a []string.
+// The flag is specified using the format: value1,value2,value3
+type StringSlice struct {
```

Comment:
We already have a `Strings` type for string slices: https://github.com/kubernetes/test-infra/blob/master/prow/flagutil/strings.go
Including this new type for comma delimited lists (instead of repeated flags) seems fine, but a name that distinguishes this type from `Strings` would be preferable since they both resolve to string slices.